src/Microsoft.Extensions.ML/Microsoft.Extensions.ML.csproj

```diff
@@ -1,17 +1,27 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoRoot)pkg/Directory.Build.props" />
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <IncludeInPackage>Microsoft.Extensions.ML</IncludeInPackage>
+    <PackageId>Microsoft.Extensions.ML</PackageId>
```

Comment:
Why do we need to set `PackageId`? It should be defaulted to the project name.

Reply:
You are correct. I saw this in one of the arcade examples shared with me and didn't realize it was the default behavior. I will go ahead and remove this from all packages that have it.